### ISSUE TITLE
HTTPS Support Not Working

### ISSUE THREAD

**Issue by commonsguy** [doppllib/doppl-gradle#13]
[SOAndroidWithDoppl-20171025b.zip](https://github.com/doppllib/doppl-gradle/files/1428934/SOAndroidWithDoppl-20171025b.zip)

I have an app that uses Retrofit to hit the Stack Exchange API, using HTTPS. When I run the Doppl-converted unit tests on iOS, I get:

```
********** Running co.doppl.so.RepositoryTest **********


********** Result for co.doppl.so.RepositoryTest **********
Failures 1
current(co.doppl.so.RepositoryTest): Https support is unavailable. Fix this by:
1) If linking with -ObjC, add -ljre_ssl to the link flags.
2) If linking without -ObjC, call JavaxNetSslHttpsURLConnection_class_() to create a compile-time dependency.
message: Https support is unavailable. Fix this by:
1) If linking with -ObjC, add -ljre_ssl to the link flags.
2) If linking without -ObjC, call JavaxNetSslHttpsURLConnection_class_() to create a compile-time dependency.
description: current(co.doppl.so.RepositoryTest)
trace: com.google.j2objc.LibraryNotLinkedError: Https support is unavailable. Fix this by:
1) If linking with -ObjC, add -ljre_ssl to the link flags.
2) If linking without -ObjC, call JavaxNetSslHttpsURLConnection_class_() to create a compile-time dependency.
	at 0x00000001034612fe com.google.j2objc.LibraryNotLinkedError.<init>() + 117
	at 0x00000001033d548c java.net.URL.getURLStreamHandler() + 1533
	at 0x00000001033d4206 java.net.URL.<init>() + 636
	at 0x0000000103131544 okhttp3.HttpUrl.url() + 36
	at 0x00000001031e7b05 retrofit2.urlsession.UrlSessionCall.runRequestCall() + 1093
	at 0x00000001031e7494 retrofit2.urlsession.UrlSessionCall.execute() + 212
	at 0x00000001031ccbac retrofit2.OkHttpCall.execute() + 1180
	at 0x00000001031b14c1 retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual() + 273
	at 0x000000010309dc03 io.reactivex.Observable.subscribe() + 115
	at 0x00000001030a4e71 io.reactivex.Observable.subscribe() + 33
	at 0x00000001031afdbd retrofit2.adapter.rxjava2.BodyObservable.subscribeActual() + 109
	at 0x000000010309dc03 io.reactivex.Observable.subscribe() + 115
	at 0x00000001030a4e71 io.reactivex.Observable.subscribe() + 33
	at 0x0000000102fdd76c io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual() + 108
	at 0x00000001030ec9a1 io.reactivex.Single.subscribe() + 97
	at 0x00000001030ed461 io.reactivex.Single.subscribe() + 33
	at 0x00000001030423bc io.reactivex.internal.operators.single.SingleMap.subscribeActual() + 108
	at 0x00000001030ec9a1 io.reactivex.Single.subscribe() + 97
	at 0x00000001030ec91b io.reactivex.Single.blockingGet() + 43
	at 0x0000000102d5340e co.doppl.so.RepositoryTest.current() + 190
	at 0x00000001035dd581 java.lang.reflect.Method.invoke:object:() + 22
	at 0x00000001035dd1aa java.lang.reflect.Method.invoke() + 333
	at 0x00000001035fb9e1 org.junit.internal.runners.model.ReflectiveCallable.run() + 19
	at 0x0000000103605766 org.junit.runners.ParentRunner.runLeaf() + 64
	at 0x0000000103605657 org.junit.runners.ParentRunner.runChildren() + 367
	at 0x0000000103605aa0 org.junit.runners.ParentRunner.run() + 85
	at 0x0000000103605657 org.junit.runners.ParentRunner.runChildren() + 367
	at 0x0000000103605aa0 org.junit.runners.ParentRunner.run() + 85
	at 0x00000001035ffba1 org.junit.runner.JUnitCore.run() + 154
	at 0x0000000102db0cf9 co.touchlab.doppl.testing.DopplJunitTestHelper.run() + 1177
	at 0x0000000102db07fc co.touchlab.doppl.testing.DopplJunitTestHelper.runSpecificTest() + 44
	at 0x0000000102db05f0 co.touchlab.doppl.testing.DopplJunitTestHelper.run() + 656
	at 0x0000000102db27c6 co.touchlab.doppl.testing.DopplJunitTestHelper.runResource() + 822
	at 0x0000000102db243f co.touchlab.doppl.testing.DopplJunitTestHelper.runResource() + 31
	at 0x0000000102db240d co.touchlab.doppl.testing.DopplJunitTestHelper.runResource() + 29
	at 0x000000010633c54d UIViewController.loadViewIfRequired() + 1235
	at 0x000000010633c99a UIViewController.view() + 27
	at 0x000000010620aae3 UIWindow.addRootViewControllerViewIfPossible() + 122
	at 0x000000010620b1eb UIWindow._setHidden:forced:() + 294
	at 0x000000010621e098 UIWindow.makeKeyAndVisible() + 42
	at 0x0000000106190521 UIApplication._callInitializationDelegatesForMainScene:transitionContext:() + 4711
	at 0x0000000106195751 UIApplication._run() + 1720
	at 0x000000010655ae00 __UICanvasLifecycleMonitor_Compatability._scheduleFirstCommitForScene:transition:firstActivation:completion:() + 924
	at 0x000000010692d840 _UICanvas._enqueuePostSettingUpdateTransactionBlock:() + 153
	at 0x000000010655a9f9 __UICanvasLifecycleMonitor_Compatability._scheduleFirstCommitForScene:transition:firstActivation:completion:() + 249
	at 0x000000010655b257 __UICanvasLifecycleMonitor_Compatability.activateEventsOnly:withContext:completion:() + 668
	at 0x0000000106ecedd9 _UIApplicationCanvas._transitionLifecycleState() + 262
	at 0x0000000106ecec92 _UIApplicationCanvas._transitionLifecycleState() + 444
	at 0x0000000106bb087e _UICanvasLifecycleSettingsDiffAction.performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:() + 420
	at 0x0000000106bb067a _UICanvasLifecycleSettingsDiffAction.performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:() + 231
	at 0x000000010692ce9d _UICanvas.scene:didUpdate() + 392
	at 0x0000000106193fd2 UIApplication.workspace:didCreateScene:withTransitionContext:completion:() + 523
	at 0x000000010676889b UIApplicationSceneClientAgent.scene:didInitialize() + 369
	at 0x000000010a24f1b4 FBSSceneImpl._didCreate() + 338
	at 0x000000010a257ca9 FBSWorkspace.client:handleCreateScene:withCompletion:() + 235
	at 0x000000010a283384 FBSSerialQueue._performNext() + 464
	at 0x000000010a2838f3 FBSSerialQueue._performNextFromRunLoopSource() + 45

Total: 1
Failures: 1
```

The attached ZIP archive has the project, complete with hand-modified podspec to try to include `jre_ssl`, with no results.

`ViewController` has the `JavaxNetSslHttpsURLConnection_class_()` call commented out. Uncommenting it results in a build error.

**Comment by commonsguy** [comment]
Note that HTTPS is required by iOS, so we need to come up with a reliable recipe for getting it working.

**Comment by commonsguy** [comment]
Seems resolved in v0.9.3 of the Doppl Plugin for Gradle -- many thanks!
